Documenting the difference betwen df[!, :col] and df[:, :col] (#2000)
JuliaData/DataFrames.jl: docs/src/man/getting_started.md

```

Columns can be directly (i.e. without copying) accessed via `df.col` or `df[!, :col]`. The latter syntax is more flexible as it allows passing a variable holding the name of the column, and not only a literal name. Note that column names are symbols (`:col` or `Symbol("col")`) rather than strings (`"col"`). Columns can also be accessed using an integer index specifying their position. 

> `Notice that, changing the elements of the array returned by `df[!, :col]` will affect the original `df`, while `df[:, :col]` returns a copy of the array, and changing it doesn't change `df` values.

Reviewer: ```suggestion
Notice that, changing the elements of the column returned by `df[!, :col]` will affect the values stored in the original `df`, while `df[:, :col]` returns a copy of the column, and changing it does not change `df`.
```

> Notice that, changing the elements of the column returned by `df[!, :col]` will affect the values stored in the original `df`, while `df[:, :col]` returns a copy of the column, and changing it does not change `df`.

Reviewer 2: AFAICT `df[:, :col]` hasn't been mentioned yet at this point. Can you rephrase this so that it introduces `:`, rather than making it sound like users already know it?

Author: Fixed in https://github.com/JuliaData/DataFrames.jl/pull/2000/commits/c1de00de14e23f0fc19aa208fbbf5c5b130068eb

Columns can be directly (i.e. without copying) accessed via `df.col` or `df[!, :col]`. The latter syntax is more flexible as it allows passing a variable holding the name of the column, and not only a literal name. Note that column names are symbols (`:col` or `Symbol("col")`) rather than strings (`"col"`). Columns can also be accessed using an integer index specifying their position. 

> Since `df[!, :col]` does not make a copy, changing the elements of the column returned by this syntax will affect the values stored in the original `df`. Use the `df[:, :col]` syntax to get a copy of the column, which can be modified without changing `df`.

Author: This is too picky, but regarding the last part, I like my sentence structure more :smile: Combining the two sentences and using `which` makes it a little vague (like: `what can be modified?`). 
Otherwise, it looks good.

Reviewer 2: Can you suggest a wording?

Author: ```suggestion
Since `df[!, :col]` does not make a copy, changing the elements of the column returned by this syntax will affect the values stored in the original `df`. 

To get a copy of the column use `df[:, :col]`. Changing the column returned by this syntax does not change `df`.
```
